### PR TITLE
Add honza and stbenjam to owners

### PR DESCRIPTION
**Pull request by stbenjam**
Also sorted alphabetically.

### CODE REVIEW

**Comment by honza** [comment]
obama

**Comment by dhellmann** [comment]
Those test failures don't look related to this.

/lgtm

File: OWNERS [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-  - russellb
   - dhellmann
-  - mhrivnak
   - hardys
+  - honza
+  - mhrivnak
+  - russellb
+  - stbenjam
   - zaneb
+  - russellb
```
Comment (hardys) [inline OWNERS:19]: you've got a duplicate trailing russellb here which needs removing

**Comment by hardys** [comment]
Looks good apart from the duplicate line - as discussed on slack I can manually apply the bug label since a bugzilla should not be required for an OWNERS file only change.

File: OWNERS (hunk 1 above)
Comment (stbenjam) [reply OWNERS]: Fixed

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**